components/ai_chat/resources/page/stories/components_panel.tsx

```diff
@@ -25,7 +25,8 @@ function getCompletionEvent(text: string): mojom.ConversationEntryEvent {
     completionEvent: { completion: text },
     pageContentRefineEvent: undefined,
     searchQueriesEvent: undefined,
-    searchStatusEvent: undefined
+    searchStatusEvent: undefined,
+    selectedLanguageEvent: { selectedLanguage: '' }
```

Comment:
Actually these need to be `undefined` since this is a union and therefore only 1 property should have a value. Just saw this when rebasing so will fix in another PR since it's not causing any errors as it's not being read in the frontend anyway.

Reply:
> so will fix in another PR
Gotcha and thanks!